extensions/wikia/WikiaInYourLang/WikiaInYourLangController.class.php

```diff
@@ -65,8 +69,17 @@ public function getNativeWikiaInfo() {
 						$oNativeWiki->city_url,
 						$oNativeWiki->city_title,
 					];
-
-					$sMessagesAry = $this->prepareMessage( $sTargetLanguage, $aMessageParams );
+					$isMainPageLink = true;
+					if ( $wgWikiaEnvironment !== WIKIA_ENV_DEV ) {
```

Comment:
Unless it breaks on devboxes for some reason, I'd still call it, even if it usually won't return a matching page. I think it's better to minimise environmental differences unless absolutely necessary.